src/Network/Google/Drive/File.hs

```diff
@@ -101,9 +102,9 @@ instance FromJSON FileData where
     parseJSON _ = mzero
 
 instance ToJSON FileData where
-    toJSON FileData{..} = object
+    toJSON FileData{..} = object $
+        (maybe [] (pure . ("modifiedDate" .=)) fileModified) ++
         [ "title" .= fileTitle
-        , "modifiedDate" .= fileModified
```

Comment:
With `fileModified` a `Maybe`, if you were to leave this code unchanged, the only difference would be getting `{ "modifiedDate": null }` in the `Nothing` case, rather than the the key being missing. If that's functionality equivalent, I think I'd prefer doing that. Do you happen to know if a `null` value would be the same as a missing key?